### PR TITLE
Release notes 4.3 updates

### PR DESCRIPTION
**Pull request by codyhoag**

### CODE REVIEW

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -129,6 +146,21 @@ the {product-title} web console.
 The `KubeletTooManyPods` alert now uses the Pod capacity metric as a threshold
 instead of a fixed number.
 
+[id="ocp-4-3-monitor-your-own-services"]
+==== Monitor your own Services (Technology Preview)
```
Comment (jboxman) [inline release_notes/ocp-4-3-release-notes.adoc]: Based on our current glossary usage for _service_, I wonder if this needs to be capitalized? (_Capitalize when referring to the Kubernetes object specifically._) Are we talking about some particular application service or specifically the Service API object? I'm guessing the former.

(https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/term_glossary.adoc#service)

File: release_notes/ocp-4-3-release-notes.adoc [hunk 2]
```diff
@@ -129,6 +146,21 @@ the {product-title} web console.
...
+
+The existing monitoring stack can be extended so you can configure monitoring
+for your own Services.
+
+[id="ocp-4-3-query-browser-available-throught-devexp-console"]
+==== Query browser available through the DevExp console (Technology Preview)
+
+Querying metrics is now available through the developer perspective inside the
```
Comment (jboxman) [inline release_notes/ocp-4-3-release-notes.adoc]: Is the _developer perspective_ what the DevExp console is? Or is it the developer tab available in the UI?

File: release_notes/ocp-4-3-release-notes.adoc [hunk 3]
```diff
@@ -722,8 +759,11 @@ Service Mesh from version 1.0.1 to 1.0.2].
...
-environments: 
+environments:
 +
-====
+----
 oc tag <jenkins_source_image> jenkins:2 --reference-policy=source -n openshift
-====
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1787112[*BZ#1787112*])
```
Comment (jboxman) [inline release_notes/ocp-4-3-release-notes.adoc]: I don't know if our format for this is immutable, but it looks odd having the BZ link dangling after the code block. I wonder if it's permissible to move it above the code block?

**Review by jboxman — APPROVED** [review]
Only a few minor comments.

File: release_notes/ocp-4-3-release-notes.adoc (hunk 1 above)
Comment (codyhoag) [reply release_notes/ocp-4-3-release-notes.adoc]: Yeah, this isn't specifically referring to the Service API object. I'll lowercase this instance of *service*.

File: release_notes/ocp-4-3-release-notes.adoc (hunk 3 above)
Comment (codyhoag) [reply release_notes/ocp-4-3-release-notes.adoc]: Yeah, I was struggling with this. I did see it done in the 4.2 release notes, which is why I moved forward with it. But I'm totally in agreeance that it looks kind of funky. I'll move it above the code block. :+1:

File: release_notes/ocp-4-3-release-notes.adoc (hunk 2 above)
Comment (codyhoag) [reply release_notes/ocp-4-3-release-notes.adoc]: It's referring to the Developer tab in the UI. I'll make this more clear.

**Pull request merged**